apps/medusa-be/.env.template

```
@@ -20,12 +22,26 @@ AUTH_CORS=http://localhost:5173,http://localhost:9000
 POSTGRES_URL=
 MEDUSA_ADMIN_ONBOARDING_TYPE=
 MEDUSA_ADMIN_ONBOARDING_NEXTJS_DIRECTORY=
-JWT_SECRET=
-COOKIE_SECRET=
-
-
 
+# PPL Fulfillment Provider
+# Set to "1" to enable PPL service, "0" to disable
+PPL_ENABLED=0
+PPL_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PPL_CLIENT_ID key should go before the PPL_ENABLED key

(UnorderedKey)